AlicerceTests/ServiceLocatorTests.swift

```diff
@@ -135,8 +326,8 @@ class ServiceLocatorTests : XCTestCase {
             
             
             // Get registered services
-            let rStringService: String = try serviceLocator.get()
-            let rDoubleService: Double = try serviceLocator.get()
+            let rStringService: String = try serviceLocator.get(name: "String")
```

Comment:
I don't agree with these `get`'s receiving a name, because the services aren't registered using a name and the caller *shouldn't* know what name is auto generated by the SL.